_component_design/pills.md

```diff
@@ -224,3 +224,28 @@ Ensure that you follow the truncation guidelines in the Ellipses section of the
 </div>
 
 </section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Too many
+When you have 10 or more pills, use the too many pill to indicate to the user that there are additional pills.
```

Comment:
QQ; Shouldn't the `too many` part be styled? It seems odd just sitting there.

Also, all of the other pill types talk simply about the state, not the pill (e.g., it's not a `hover pill` or an `active pill`, it's a `hover state` or an `active state`).